docs/pages/contributing.rst

```diff
@@ -49,6 +49,48 @@ current state.
 Development version
 -------------------
 
+Earthly Setup
+^^^^^^^^^^^^^
+
+You can build turbodbc and run the full test suite with `earthly <https://earthly.dev>`_.
+
+See the install instructions on how to get earthly: `https://earthly.dev/get-earthly <https://earthly.dev/get-earthly>`_.
+
+Build turbodbc with the default arguments:
+
+::
+
+    earthly --allow-privileged +build
```

Comment:
What is the "net result" of this build?
I assume this includes a python package (wheel and/or sdist?)? Where will it be (I could not find it locally, although from reading over the earth files, I though it should be somewhere on the host).

Reply:
Updated the earthly docs to represent this.